src/test/ui/error-codes/E0268.stderr

```diff
@@ -2,7 +2,10 @@ error[E0268]: `break` outside of a loop
   --> $DIR/E0268.rs:2:5
    |
 LL |     break;
-   |     ^^^^^ cannot `break` outside of a loop
+   |     ^^^^^
+   |     |
+   |     cannot `break` outside of a loop
+   |     help: consider removing the: `break`
```

Comment:
We have a similar issue here, we'd want to include the trailing `;` in the removal suggestion (although a bare `;` isn't the end of the world) as well *and* the result of applying this change is no change (we are replacing `break` with `break`).

Reply:
So I need to implement a way to check if there is a ; and then display that. Got it.

Reply:
Not display it, but extend the span to cover it. Thinking about it a bare `;` will be follow up error, so this suggestion isn't _terrible_, you just have to change the code to actually remove `break` instead of replacing it with itself.